tests/test_peak_core_and_tile.py

```diff
@@ -0,0 +1,197 @@
+import glob
```

Comment:
Can you also move this to `test_peak_core` folder?

Reply:
Sure, I wasn't sure if that would be the best place since it technically tests the core and the tile, but I don't really care that much